src/govuk/components/notification-banner/template.njk

```diff
@@ -0,0 +1,40 @@
+{% if params.role %}
+  {% set role = params.role %}
+{% elif params.type == "success" or params.type == "error" %}
+  {% set role = "alert" %}
+{% else %}
+  {% set role = "region" %}
+{% endif %}
+
+{% if params.tabindex %}
+  {% set tabindex = params.tabindex %}
+{% elif params.type == "success" or params.type == "error" %}
+  {% set tabindex = "-1" %}
+{% endif %}
+
+{% set titleHeadingLevel = params.titleHeadingLevel if params.titleHeadingLevel else 2 %}
+
+<div class="govuk-notification-banner
+  {%- if params.type == "success" %} govuk-notification-banner--success{% endif %}
+  {%- if params.type == "error" %} govuk-notification-banner--error{% endif %}
```

Comment:
Could we simplify this to something like:

```
{%- if params.type %} govuk-notification-banner--{{params.type}}{% endif %}
```

It does mean if someone passes in something other than `success` or `error` then it will generate an unused class, but not sure that's an issue?

Reply:
I don't think we should add a redundant class but I've tried to simplify this logic and moved it to the top of the template so it's now not cluttering the markup, have a look.